main.py

```diff
@@ -369,3 +370,20 @@ def enable_lldp(self):
         msg_error = "Some interfaces couldn't be found and activated: "
         return jsonify({msg_error:
                         error_list}), 400
+
+    @rest('v1/time', methods=['GET'])
+    def get_time(self):
+        """Get LLDP polling time."""
+        return jsonify({"Polling time in seconds": self.polling_time}), 200
+
+    @rest('v1/time/<time_sec>', methods=['POST'])
+    def set_time(self, time_sec):
+        """Set LLDP polling time."""
```

Comment:
Please add comment or log warning the user that this change will not be saved permanently.

Reply:
Thanks for the suggestion @hdiogenes, I added a log warning indicating that this change is not persistent.
``` python
log.info("Polling time has been updated, but this change will not be saved permanently.")
```